front-end/src/components/newAccountForm.tsx

```diff
@@ -0,0 +1,188 @@
+      borderRadius: "8px",
+      padding: "25px 75px",
+    } as const,
+    input: {
+      margin: "10px",
+    },
+    button: {
+      backgroundColor: "#D64309",
+    },
+  };
+
+  const [firstNameField, setFirstNameField] = useState("");
+  const [lastNameField, setLastNameField] = useState("");
+  const [emailField, setEmailField] = useState("");
+  const [passwordField, setPasswordField] = useState("");
+  const [passwordField2, setPasswordField2] = useState("");
+  const [alertReason, setAlertReason] = useState("");
+
+  const handleFirstNameChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setFirstNameField(e.target.value);
+  };
+
+  const handleLastNameChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setLastNameField(e.target.value);
+  };
+
+  const handleEmailChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setEmailField(e.target.value);
+  };
+
+  const handlePasswordChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setPasswordField(e.target.value);
+  };
+
+  const handlePassword2Change = (e: ChangeEvent<HTMLInputElement>) => {
+    setPasswordField2(e.target.value);
+  };
+
+  //Alpha Characters only (case insensitive)
+  const checkSignage = /^[a-z]+$/i;
+
+  //Basic email signage.
+  const checkEmailSignage = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i
+
+  /* min eight characters,
+  max 20 characters, 
+  at least one letter, one number 
+  and one special character */
+  const checkPassSignage = /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&^])[A-Za-z\d@$!%*#?&^]{8,20}$/;
+
+  const handleSubmit = () => {
+    console.log(`First Name: ${firstNameField}`);
+    console.log(`Last Name: ${lastNameField}`);
+    console.log(`Email: ${emailField}`);
+    console.log(`Password: ${passwordField}`);
+    console.log(`Check Password: ${passwordField2}`);
+
+    //All fields must be filled
+    if (firstNameField  === "" || lastNameField  === "" || emailField === "" || passwordField === "" || passwordField2 === "") {
+      setAlertReason("Enter your name, email address, and password.");
+      return;
+    }
+
+    //Check username/password requirements and security for text fields.
+    //Firstname and Lastname forms must use appropriate signage (alpha characters only)
+    if (!(firstNameField.match(checkSignage)) || !(lastNameField.match(checkSignage)) ) {
+      setAlertReason("Name fields must only be alphabetic characters.");
+      return;
+    }
+
+    //Email should only allow basic signage.
+    if (!(emailField.match(checkEmailSignage)) ) {
+      setAlertReason("Please enter a valid email address.");
+      return;
+    }
+
+    //Passwords must match
+    if (passwordField !== passwordField2) {
+      setAlertReason("Your passwords must match.");
+      return;
+    }
+
+    //Passwords must be correct format (8-20 characters, 1 letter, 1 number, 1 special character, no spaces)
+    if (!(passwordField.match(checkPassSignage)) || !(passwordField2.match(checkPassSignage)) ) {
```

Comment:
Don't need this or, we've already determined passwordField === passwordField2 by this point

Reply:
Changed the check to "!(passwordField.match(checkPassSignage))"